packages/app-config-writer/src/prompt/smartlinks-config.ts

```diff
@@ -146,7 +146,7 @@ export async function promptUserPass(log?: ToolsLogger): Promise<AxiosBasicCrede
                 name: 'password',
                 message: `${cyan(t('info.password'))}`,
                 validate: (value: string): boolean | string => {
-                    if (!value || !value.trim()) {
+                    if (!value?.trim()) {
```

Comment:
Are these modifications aimed at addressing linting errors? They aren't associated with the version upgrade. While they are necessary, I'm mindful of maintaining a distinction between the version upgrade and code clean-up.

Reply:
@ullasholla I know what you mean, I hadn't planned to update it originally. 
The @typescript-eslint/eslint-plugin version needed to be upgraded as the old one didn't support the new TypeScript version.